species/plot/plot_retrieval.py

```diff
@@ -296,6 +299,24 @@ def plot_pt_profile(
             tau = pressure * 1e6 * 10.0 ** item[param_index["log_delta"]]
             temp = (0.75 * item[param_index["tint"]] ** 4.0 * (2.0 / 3.0 + tau)) ** 0.25
 
+        elif pt_profile == "gradient":
+            num_layer = 6 # could make a variable in the future
+            layer_pt_slopes = np.ones(num_layer) * np.nan
+            for index in range(num_layer):
+                layer_pt_slopes[index] = item[param_index[f'PTslope_{num_layer - index}']]
+
+            try:
+                from petitRADTRANS.physics import dTdP_temperature_profile
+            except ImportError:
+                print("""Can\'t import the dTdP profile function from petitRADTRANS,
```

Comment:
Add the `Raise` also here? Sorry I had missed this previously. Thanks for already having updated `plot_pt_profile`